finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,167 @@
+// function frogCountDown() {
+//     // for loop to go through 10 to 1 count down
+//     // consider special cases of 3, 2, 1
+//
+//     for (let i = 10; i > 0; i--) {
+//         if (i > 2) {
+//             console.log(`${i} speckled frogs sat on a log eating some most delicious bugs.`);
+//             console.log(`One jumped in the pool where it's nice and cool, then there were ${i-1} speckled frogs.`);
```

Comment:
Impressive use of backticks here to be able to use apostrophes on the contractions!